docs/manual/source/install/index.html.md.erb

```diff
@@ -24,8 +24,8 @@ limitations under the License.
 It is **very important** to meet the minimum version of the following
 technologies that power Apache PredictionIO®.
 
-* Apache Hadoop 2.6.5 (optional, required only if YARN and HDFS are needed)
-* Apache Spark 1.6.3 for Hadoop 2.6
+* Apache Hadoop 2.7.3 (optional, required only if YARN and HDFS are needed)
+* Apache Spark 2.1.1 for Hadoop 2.7
```

Comment:
I wonder if we should put Hadoop 2.6.x/2.7.x, and Spark 2.0+ here. We still run CI on Spark 2.0, and the Hadoop requirement comes mostly from Spark.